moveit_planners/chomp/chomp_interface/include/chomp_interface/chomp_planning_context.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * chomp_planning_context.h
+ *
+ *  Created on: 27-Jul-2016
+ *      Author: ace
```

Comment:
Sorry for these errors. Correcting them now. I was very bent on getting CHOMP to work that I really didn't "clean out" the packages I imported. 
Who should I attribute the copyright to? Myself or ?